.github/workflows/deploy.yml

```diff
@@ -3,21 +3,23 @@ name: Build and Deploy
 on:
   push:
     branches:
-      - main
+      - SOW-24/wrap-form-in-app
     paths:
       [
-        ".github/workflows/storybook.yml",
+        ".github/workflows/deploy.yml",
+        "vite.config.ts",
         "lib/stories/**",
         "lib/**/*",
+        "src/**/*",
         "package-lock.json",
       ]
```

Comment:
We're probably at the point where we're not really getting any benefit from these paths, maybe we should just get rid of them?

Reply:
That's basically already happening anyway and the files that aren't included here aren't changing very often